tools/serverpod_cli/test/analyzer/models/stateful_analyzer/model_validation/field_validation_test.dart

```diff
@@ -293,7 +294,10 @@ void main() {
         expect(collector.errors, isNotEmpty,
             reason: 'Expected an error but none was generated.');
 
-        var error = collector.errors.first;
+        var error = collector.errors.first as SourceSpanSeverityException;
+
+        expect(error.severity, SourceSpanSeverity.info);
```

Comment:
Question: Is this just a smoke test or should it be added in more places?

Reply:
I think we have similar tests for any error that is on the info level.